### PR TITLE
#162412649 user create incident 

### PR DESCRIPTION
**Pull request by bl4ck4ndbr0wn**
#### What does this PR do?
Have CRUD in creating resource endpoint to be able to create a new incident record.
#### Description of Task to be completed?
Test api endpint user can create a new incident.
create api endpoint.
#### How should this be manually tested?
Clone the repo as bellow:
 ```
 git clone - b ft-user-create-incident-162412649 git@github.com: bl4ck4ndbr0wn/iReporter.git
```
Create a virtual environment and activate it
 ```
 python3 - m venv env
 source .env
 ```
 Install all the dependencies needed by
 ```
 pip install - r requirements.txt
 ```
 Run the flask application
 ```
  flask run
 ```
ON postman request:
* POST  /red-flags
* :params : {
  “id” : Integer,
  “createdOn” : Date,  
  “createdBy” : Integer, // represents the user who created this record
  “type” : String,       // [red-flag, intervention]
  “location” : String,   // Lat Long coordinates
  “status” : String,     // [draft, under investigation, resolved, rejected]
  “Images” : [Image, Image], 
  “Videos” : [Image, Image],
  “comment” : String
   ...
}

#### What are the relevant pivotal tracker stories?
 [  # 162412649](https://www.pivotaltracker.com/n/projects/2226962)
#### Questions:

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,16 +1,8 @@
 from flask import Flask
 from flask_restful import Api
-from flask_jwt_extended import JWTManager
+from flask_cors import CORS
```
Comment (thoth-ky) [inline app/__init__.py:2]: why are you using CORS?

File: app/tests/v2/system/test_records.py [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
 
 class RecordTest(BaseTest):
-    pass
+    def test_create_new_incident(self):
+        """
+        Test create new incident POST Endpoint.
+        :return: status code 201 (Created)
+        """
+        r = self.create_incident()
+        expected = {"status": 201,
```
Comment (thoth-ky) [inline app/tests/v2/system/test_records.py:12]: Let's expect that when a new incident is created, the user receives all the details about it instead of just ID.

**Review by thoth-ky — CHANGES_REQUESTED** [review]
Add more tests mocking a user sending `POST` requests to create incidents. This should also have different edge cases such as trying to create incidents with invalid data

File: app/tests/v2/system/test_records.py (hunk 2 above)
Comment (bl4ck4ndbr0wn) [reply app/tests/v2/system/test_records.py:12]: ok.

File: app/__init__.py (hunk 1 above)
Comment (bl4ck4ndbr0wn) [reply app/__init__.py:2]: To allow access by other domains  like GitHub pages
